tactics/ind_tables.mli

```diff
@@ -47,26 +53,30 @@ type individual_scheme_object_function =
     inductive's name.
 *)
 
-val declare_mutual_scheme_object : string ->
-  ?suff:string ->
-  ?deps:(Environ.env -> MutInd.t -> scheme_dependency list) ->
+val declare_mutual_scheme_object : string list * UnivGen.QualityOrSet.t option ->
+  (Declarations.one_inductive_body option -> string) ->
+  ?deps:(Environ.env -> Names.MutInd.t -> bool -> scheme_dependency list) ->
   mutual_scheme_object_function -> mutual scheme_kind
 
-val declare_individual_scheme_object : string ->
-  ?suff:string ->
-  ?deps:(Environ.env -> inductive -> scheme_dependency list) ->
+val declare_individual_scheme_object : string list * UnivGen.QualityOrSet.t option ->
+  (Declarations.one_inductive_body option -> string) ->
+  ?deps:(Environ.env -> inductive -> bool -> scheme_dependency list) ->
   individual_scheme_object_function ->
   individual scheme_kind
 
-val is_declared_scheme_object : string -> bool
+val is_declared_scheme_object : Key.t -> bool
 (** Is the string used as the name of a [scheme_kind]? *)
 
-val scheme_kind_name : _ scheme_kind -> string
+val scheme_kind_name : _ scheme_kind -> Key.t
 (** Name of a [scheme_kind]. Can be used to register with DeclareScheme. *)
 
+val scheme_key : Key.t -> _ scheme_kind
```

Comment:
this is incorrect, it doesn't respect the meaning of the phantom type argument of scheme_kind